py2-numpy.spec

```diff
@@ -39,3 +40,8 @@ sed -ideleteme 's|#!.*/bin/python|#!/usr/bin/env python|' \
   %{i}/lib/python*/site-packages/numpy-*/numpy/distutils/conv_template.py
   
 find %{i} -name '*deleteme' -delete
+mkdir %{i}/c-api
+PYTHONV=$(echo $PYTHON_VERSION | cut -f1,2 -d.)
+OSARCH=$(uname -m)
+[ -d  %{i}/${PYTHON_LIB_SITE_PACKAGES}/numpy-%{realversion}-py${PYTHONV}-linux-$OSARCH.egg/numpy/core ] || exit 1
+ln -s   ../${PYTHON_LIB_SITE_PACKAGES}/numpy-%{realversion}-py${PYTHONV}-linux-$OSARCH.egg/numpy/core %{i}/c-api/core
```

Comment:
We created symlink in numpy tool itself to catch any change in directory structure which the build of numpy itself.